src/shared/components/oncoprint/DeltaUtils.ts

```diff
@@ -242,7 +242,11 @@ export function transitionSortConfig(
         nextSortConfig.type === "order" &&
         (prevSortConfig.order !== nextSortConfig.order))
         ||
-        !_.isEqual(prevSortConfig, nextSortConfig)) {
+        !_.isEqual(prevSortConfig, nextSortConfig)
+    ) {
+        if (nextSortConfig.type === "cluster") {
+            oncoprint.removeAllExpansionTracksInGroup(nextSortConfig.track_group_index);
```

Comment:
@adamabeshouse, TypeScript claims that this method does not exist, even though it's defined here: https://github.com/cBioPortal/oncoprintjs/blob/v1.0.79/src/js/oncoprint.js#L551

Could you point me at where I should declare the type for it?

Reply:
ah, yes you'll need to add it into index.d.ts in the home directory of oncoprintjs

Reply:
Done in cBioPortal/oncoprintjs#59

Reply:
released `v1.0.83`